nemo/nemo/utils/argparse.py

```diff
@@ -73,8 +73,8 @@ def __init__(self, **kwargs):
                                "specify either num_epochs or max_steps")
         self.add_argument("--lr", type=float, default=1e-3,
                           help="base learning rate")
-        # self.add_argument("--lr_policy", type=str,
-        #                   help="learning rate decay policy")
+        self.add_argument("--lr_policy", type=str, default='WarmupAnnealing',
```

Comment:
Any reasons to uncomment this?

Reply:
isn't lr_policy a common parser argument?

Reply:
Let's comment it back out